config/sets/phpunit100.php

```diff
@@ -29,6 +29,10 @@
         // https://github.com/sebastianbergmann/phpunit/issues/4087
         new MethodCallRename('PHPUnit\Framework\Assert', 'assertRegExp', 'assertMatchesRegularExpression'),
 
+        // https://github.com/sebastianbergmann/phpunit/issues/5220
+        new MethodCallRename('PHPUnit\Framework\Assert', 'assertObjectHasAttribute', 'assertObjectHasProperty'),
+        new MethodCallRename('PHPUnit\Framework\Assert', 'assertObjectNotHasAttribute', 'assertObjectHasNotProperty'),
```

Comment:
the method seems should be `assertObjectNotHasProperty`, see https://github.com/sebastianbergmann/phpunit/blob/bb24b2746512bc8be5122bb220064df8f8687451/src/Framework/Assert.php#L1052C34-L1052C60

Reply:
https://github.com/rectorphp/rector-phpunit/pull/340